<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackagesWidget</class>
 <widget class="QWidget" name="PackagesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="managersTabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="aptTab">
      <attribute name="title">
       <string>APT</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="AptManagerWidget" name="aptManagerWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rpmTab">
      <attribute name="title">
       <string>RPM</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="RpmManagerWidget" name="rpmManagerWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="repoTab">
      <attribute name="title">
       <string>Repo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="RepoManagerWidget" name="repoManagerWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AptManagerWidget</class>
   <extends>QWidget</extends>
   <header>ui/managerwidgets/aptmanagerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RpmManagerWidget</class>
   <extends>QWidget</extends>
   <header>ui/managerwidgets/rpmmanagerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RepoManagerWidget</class>
   <extends>QWidget</extends>
   <header>ui/managerwidgets/repomanagerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
